packages/base-nnp/base-nnp.base/opam

```diff
@@ -0,0 +1,10 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+synopsis: "Naked pointers prohibited in the OCaml heap"
+description: """
+Installed when the compiler does not permit naked pointers in
+the heap. Prior to OCaml 5.00.0, this mode was expressly selected
+by configuring with --disable-naked-pointers. The shared memory
+parallelism added in OCaml 5.00.0 requires this mode.
+"""
+depends: "ocaml" {>= "5.00"} | "ocaml-option-nnp" | "ocaml-variants" {= "4.06.1+no-naked-pointers+flambda"}
```

Comment:
```suggestion
depends: "base-domains" | "ocaml-option-nnp" | "ocaml-variants" {= "4.06.1+no-naked-pointers+flambda"}
```

Reply:
What about adding `ocaml-option-nnpchecker` as well. I know it’s not really "requiring" no-naked-pointers but if you want to constrain against one, then the other should also probably come with it.

Reply:
I don't understand what you're suggesting here?

Reply:
Something like that:
```suggestion
depends: "base-domains" | "ocaml-option-nnp" | "ocaml-option-nnpchecker" | "ocaml-variants" {= "4.06.1+no-naked-pointers+flambda"}
```

Reply:
That's a contradiction - `ocaml-option-nnpchecker` and `ocaml-option-nnp` can't be installed at the same time? The NNP checker only works if you're _in_ no-naked-pointers mode.

Reply:
yes, I know. Internally it’s not in no-naked-pointers mode, but for users it’s the same thing given if it doesn’t work with one, it won’t work with the other either.